### DISCUSSION TITLE
Summaries on Docker from CNCFStudent's #1 Event

### DISCUSSION THREAD

**Discussion by verma-kunal** [EddieHubCommunity/support#1799] [Show and tell]
This thread contains the summaries on **Docker**, which were written by our **community members as** a part of the 1st event of **CNCFStudents** + **DoK Community** + **CivoCloud**!
The session was conducted by Jeff Carpenter on "Introduction on Docker & OpenSource" & then all the attendees were given a task to write a summary of the session! The best ones would be awarded with swags & the winner's summary would be put up on **Dok Community's** YT channel with the voice-over of the winner!

Winners:
1st Prize - @kaiwalyakoparkar 
Prizes won in the 2nd category - @verma-kunal , @rohan-kulkarni-25 , @ashwinexe

**Comment by kaiwalyakoparkar** [comment]
Thanks, @verma-kunal for starting this discussion. I will share the video with my voice over and my summary soon :)

**Reply by verma-kunal** [reply]
Sure! No problem 👍

**Comment by ashwinexe** [comment]
I'll go first, here's my summary on Docker :

Hi my name is Ashwin and today I'm gonna tell you about the Docker. If you don't know anything about Docker, worry not this is the perfect place for you to **get started**. You must have heard Docker or to Dockerize something as a **buzzword** in projects or organizations, Docker is a set of tools to build, ship and run any software with all its dependencies at ease. Or in English, it’s a container which contains the product so when you share it with your friends or colleagues the stuff inside the container is not lost and others get the same experience of product as you do. But **why docker?** Once upon a time in an ideal world, Sun Microsystems introduced JAVA and its ability to write once run anywhere. Well, Docker can do that, but with **any language or tech stack!**
Let’s see how Docker does that. It starts by creating an image (just like image or photo is a snapshot of real world) image here is snapshot of instructions, then the image can be used to create multiple containers. But wait, **What’s a container?** If image is a cookie cutter, then cookies made by the cutter are the containers! Well, that was easy. A Lot of people are confused with docker and a Virtual Machine (VM). The main difference here is that docker only runs the processes needed by the containers in it, saving a lot of space and gaining speed, efficiency and cost effectiveness!
That was Docker-made-easy for you!

**Comment by verma-kunal** [comment]
**My summary on Docker**
P.S it came to of more length than that was required !!
Link -> https://bit.ly/3x0vgTp

**Reply by ashwinexe** [reply]
That's very well structured and detailed Kunal 😃

**Reply by verma-kunal** [reply]
Thank you 🙂
Yours was really informative in the beginner's point of view, kudos to the efforts 👍

**Reply by ashwinexe** [reply]
> Thank you 🙂
> Yours was really informative in the beginner's point of view, kudos to the efforts 👍

Haha yes, I myself being beginner in Docker, poured my first impressions in the summary

**Comment by eddiejaoude** [comment]
Wow great work 💪

**Reply by verma-kunal** [reply]
Thank you Eddie! Would really love some feedbacks/suggestions in the articles from your side as well !! 😇

**Reply by ashwinexe** [reply]
Thanks Eddie 😃
